docs/data-sources/project.md

```diff
@@ -71,4 +71,30 @@ The following attributes are exported:
 
 * `remove_source_branch_after_merge` - Enable `Delete source branch` option by default for all new merge requests
 
-* `packages_enabled` - Enable packages repository for the project.
\ No newline at end of file
+* `packages_enabled` - Enable packages repository for the project.
+
+* `push_rules` Project push rules (GitLab Starter/Bronze/EE only). Push Rules documented below.
+
+Push Rules (`push_rules`) is exported with the following attributes:
```

Comment:
Can we push that to a dedicated section?

Reply:
I changed this line into a header. Is is better?

Reply:
Updated to use the "Nested Blocks" format.